lib/hal/Makefile

```diff
@@ -0,0 +1,22 @@
+HAL_SRCS := \
+	$(NXDK_DIR)/lib/hal/audio.c \
+	$(NXDK_DIR)/lib/hal/fileio.c \
+	$(NXDK_DIR)/lib/hal/input.c \
+	$(NXDK_DIR)/lib/hal/io.c \
+	$(NXDK_DIR)/lib/hal/led.c \
+	$(NXDK_DIR)/lib/hal/video.c \
+	$(NXDK_DIR)/lib/hal/xbox.c
+
+HAL_OBJS = $(addsuffix .obj, $(basename $(HAL_SRCS)))
+
+$(NXDK_DIR)/lib/libnxdkhal.lib: $(HAL_OBJS)
```

Comment:
I'm personally not a big fan of stacking acronyms; I'd suggest libnxdk_hal.lib and libnxdk_usb.lib instead.

Reply:
Done.